### PR TITLE
Create draft for Installation Profiles for Drupal 8

### LINKED ISSUE #938: New Doc Proposal: Describe how to write an installation profile for Drupal 8 on Pantheon.
Some details still need to be worked out.  See pantheon-systems/drops-8#65

### PR DESCRIPTION
**Pull request by rachelwhitton**
Closes #938
Related to pantheon-systems/drops-8#65
WIP

This draft provides the required files for writing an install profile on d8. Work on hold until a Pantheon install profile for drops-8 is confirmed.

### CODE REVIEW

**Comment by greg-1-anderson** [comment]
There are still trade-offs here, so it is undecided whether or not there will be a Pantheon Profile for D8.

File: source/docs/articles/drupal/installation-profiles-on-drupal-8.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Installation Profiles on Drupal 8
+description: Learn how to write a Drupal 8 installation profile.
+category:
+  - developing
+  - drupal
+keywords: drupal 8, installation profile drupal 8, installation profile, install profile, info.yml, profile, install
+---
+Installation profiles give developers the power to sprinkle in extra installation steps for use-specific sites, such as complex module configs or content types. Writing installation profiles for Drupal 8 requires <code><em>profilename</em>.info.yml</code> and <code><em>profilename</em>.profile</code> to be placed within the `/profiles` directory. An optional <code><em>profilename</em>.install</code> file can be created in the same directory to implement `hook_install()`. Check out the [standard profile on pantheon-systems/drops-8](https://github.com/pantheon-systems/drops-8/tree/master/core/profiles/standard) as an example.
```
Comment (ari-gold) [inline source/docs/articles/drupal/installation-profiles-on-drupal-8.md]: sprinkle? (word choice)

**Comment by ari-gold** [comment]
Link to https://www.drupal.org/node/2210443 ?

**Comment by greg-1-anderson** [comment]
It is probably a good idea to explicitly say that their profile must be based on Drops-8.  It might sometimes be the case that someone wants to take an existing profile from drupal.org and run it on Pantheon.  In that case, there will be certain things they need to do to make that work.

The exact instructions on how to do this has not been finalized yet.  Today, it is sufficient to copy settings.php and settings.pantheon.php from drops-8, and put it in your site.  A better way, though, would be to make a site based on drops-8, and add & commit the installation profile on top of that.  Those instructions are likely to work regardless of how things change in the future.

**Comment by bmackinney** [comment]
Closed for more information. Please discuss at #938

**Pull request closed without merging**